gammapy/estimators/map/core.py

```diff
@@ -1084,6 +1085,70 @@ def slice_by_idx(self, slices):
             gti=self.gti,
         )
 
+    def slice_by_coord(self, slices):
+        """Slice flux maps by coordinate values
+
+        Parameters
+        ----------
+        slices : dict
+            Dict of axes names and `astropy.Quantity` or `astropy.Time` or `slice` object pairs.
+            Contains one element for each non-spatial dimension. For integer indexing the
+            corresponding axes is dropped from the map. Axes not specified in the
+            dict are kept unchanged.
+
+
+        Returns
+        -------
+        flux_maps : `FluxMaps`
+            Sliced flux maps object.
+        """
+
+        idx_intervals = []
+
+        for key, interval in zip(slices.keys(), slices.values()):
+            imin = self.geom.axes[key].coord_to_idx(interval.start)
+            imax = self.geom.axes[key].coord_to_idx(interval.stop)
+
+            idx_intervals.append(slice(imin, imax))
+
+        return self.slice_by_idx(dict(zip(slices.keys(), idx_intervals)))
+
+    def slice_by_time(self, time_min, time_max):
```

Comment:
The two methods `slice_by_time` and `slice_by_energy` are each only one line of code. They are not tested also.

I think it is better to have them as example usage in the docstring of `slice_by_coord`.